lib/rules/padding-line-between-statements.js

```diff
@@ -272,17 +272,33 @@ function verifyForNever(context, _, nextNode, paddingLines) {
  * @param {ASTNode} nextNode The next node to check.
  * @param {Array<Token[]>} paddingLines The array of token pairs that blank
  * lines exist between the pair.
+ * @param {number} blankLine Allowed number of blank lines.
+ * @param {number} foundBlankLine Number of blank lines found.
  * @returns {void}
  * @private
  */
-function verifyForAlways(context, prevNode, nextNode, paddingLines) {
+function verifyForAlways(context, prevNode, nextNode, paddingLines, blankLine, foundBlankLine) {
```

Comment:
Technically, it is not a good practice to have the too many arguments. Any ways to avoid this?

Reply:
I wanted to minimize changes and maintain backward compatibility. The more edits you make the higher the probability of errors (:
If this is critical I can work on improving it